File: unused/obfuscation.py
import random
def cipher(inp: str, cipher: list):
    index = 0
    out = list(inp)
    for i in inp:
        out[index] = chr(cipher.index(ord(i)))
        index += 1
    return ''.join(out)
def uncipher(inp: str, cipher:list):
    index = 0
    out = list(inp)
    for i in inp:
        out[index] = chr(cipher[ord(i)])
        index += 1
    return ''.join(out)
def Gcipher(key: int):
    cipher = []
    for i in range(255):
        cipher.append(i + 1)
    for s in range(25):
        index = 0
        for i in cipher:
            index2 = abs(((index^2) % 256) - s)
            index1 = abs(((index * key) - s) % 256)
            item1 = cipher.pop(index1)
            cipher.insert(index2, item1)
            item2 = cipher.pop(index2 - 1)
            cipher.insert(index1, item2)
            index += 1
    return cipher
ciphed = Gcipher(70)
#ciphed.sort()
f = open("out.dll", "wb")
lines = []
length = 2048
jarmble = b""
for i in range(length):
    jarmble += cipher(i, ciphed)

File: unused/obfuscation.c
#include <stdio.h>
#include <strings.h>
#include <stdlib.h>
#include <math.h>
int* Gcipher(unsigned long in) {
    int* cipher = (int*)malloc(256 * sizeof(int) + 1);
    cipher[255] = 0;
    for (int i = 0; i < 256; i++) {
        cipher[i] = i - 128;
    }
    for (unsigned char i = 0; i < 25; i++) {
        for (int v = 0; v < 256; v++) {
            int index1 = abs((int)pow(v, 2) % 256 - i);
            int index2 = abs(v * in - i % 256);
            int item1 = cipher[index1];
            int item2 = cipher[index2];
            cipher[index1] = item2;
            cipher[index2] = item1;
        }
    }
    return cipher;
}
int main() {
    int* ciphed = Gcipher(25);
    printf("[");
    for (int i = 0; i < 255; i++) {
        printf("%d, ", ciphed[i]);
    }
    printf("%d]\n", ciphed[25]);
}